### PR TITLE
Thai and wallet icons

### PR DESCRIPTION
**Pull request by jawad-eGHL**
** Checklist **
- [✓] All icons have a corresponding entry in `db/payment_icons.yml`
- [✓] I have followed the icon guidelines detailed in the [CONTRIBUTING.md](https://github.com/activemerchant/payment_icons/blob/master/CONTRIBUTING.md) file
- [✓] I have optimized the icon with [SVGO](https://jakearchibald.github.io/svgomg/)
- [✓] I am confident that all icons are clear and easy to read/understand
- [✓] I have provided a link to the brand icon’s brand guidelines whenever possible.

https://www.bangkokbank.com/en
https://www.cimbthai.com/en/personal/index.html
https://kasikornbank.com/en/personal
https://krungthai.com/en/personal
https://www.thanachartbank.co.th/TbankCMSFrontend/defaulten.aspx
https://www.uob.co.th/investor-en/aboutUOB/about-us-uob-corporate-profile.page
https://en.wikipedia.org/wiki/Big_C
https://www.myboost.com.my/
https://www.doku.com/en/
https://www.ecpay.com.ph/
https://www.maybank2u.com.my/maybank2u/malaysia/en/personal/services/digital_banking/maybank_qrpay_consumer.page
http://newbackend.mcash.my/
https://pay.line.me/portal/th/main
https://en.wikipedia.org/wiki/Tesco_Lotus
https://www.touchngo.com.my/

### CODE REVIEW

**Comment by jawad-eGHL** [comment]
@tauthomas01 Please review.
Please note that this commit I have made after pulling and merging the master branch of original repo into th master branch of my forked repo.
Also, my previous 2 PRs were rejected by Travis CI/CD. So this time I have re-added the files from both of those PRs.
https://github.com/activemerchant/payment_icons/pull/381/
https://github.com/activemerchant/payment_icons/pull/382/
So I will remove those PRs now

File: app/assets/images/payment_icons/tescolotus.svg [hunk 1]
```diff
@@ -0,0 +1 @@
...1LAAUAMalA+C0PSaqHtPxBSZmkPXyEwEL7LFkHRqKH1S1rInzQFxStWOTx/owEHERERERHRjahwiYq53OFDDJrwkE+c1VWWLjG+sSY8JN9ZuOFIYGpGB2VYxEBndWYKfBIWu1cUbgDlW7gKAI5XVGflC3vhBgAEpmYElS19qWpoUuhoe9iy5qgWW2E64KCTJN3tpKyzkKXS5qCiry+84z8BPEt3KZY1GmgHDkHJps2AbD/7lLKzKww3iIiIiIiIbrVKz+CwogPwPgCts0IrbgCGArDYJcAZRzf6JtrY6LcNaSkfwX5gc9JzWNzj6pg4m2agzuj37vbKHT5kNYAHHZRc9BwWF6WOidvn4LwF/d7dYu7wIWMATEDpEhp7Mj2HxfVWx8Q5naGhjY1+y5CWYi90+tNzWNzz6pg4my1n6b/J3gwO1YPdUWPG/wEQkPNIL8g6y11VPUfEwCN6ICCUvm2KlixH4ZyvLGqUHcLgNXkCxMBAyMXFMJ7+GyWbt6Dk59WQCyrO7DiDg4iIiIiIbiVTwLEKgMXOH3eQpYGpGYOdFRHdzewFHLcbAw4iIiIiIrqVRAAITM2w2Q72TsFwg4iIiIiIiIicKV/SEZiaISjDImZXVHyLzXa2NIWIyqhUzipuLYUiQ/DyqvamxERERERERI4Isp0mgprwkJEAngcQYHPy5roKYHlgasZcZ4X0/+3bMSvEcQDGcfenLM5lICy8CN1+q5u8gRsMivJ6lEFRt1oMUsdrYXB3GU9dEcdoUb/ljx59PvPzCr71wJfnw+PL18HdXmn3WxbaOyet/vlRaQcAAFCXbwMHkOV9NF6Z9PZPZ/cPf343qzbWr5Yvzg7mt7fGpS0AAEBdBA74Jz6m0+rl+qYzexxulrY/pVpbfVrs7t42mktvpS0AAECdBA4AAAAgXjUHAAAAEE7gAAAAAOIJHAAAAEA8gQMAAACIJ3AAAAAA8QQOAAAAIJ7AAQAAAMQTOAAAAIB4AgcAAAAQT+AAAAAA4gkcAAAAQDyBAwAAAIgncAAAAADxBA4AAAAgnsABAAAAxBM4AAAAgHgCBwAAABBP4AAAAADiCRwAAABAPIEDAAAAiCdwAAAAAPEEDgAAACCewAEAAADEEzgAAACAeAIHAAAAEE/gAAAAAOIJHAAAAEC8TwWyRAxRSHvjAAAAAElFTkSuQmCC"/></svg>
```
Comment (tauthomas01) [inline app/assets/images/payment_icons/tescolotus.svg:1]: This one also

image

File: app/assets/images/payment_icons/touchngo.svg [hunk 2]
```diff
@@ -0,0 +1 @@
...1LAAUAMalA+C0PSaqHtPxBSZmkPXyEwEL7LFkHRqKH1S1rInzQFxStWOTx/owEHERERERHRjahwiYq53OFDDJrwkE+c1VWWLjG+sSY8JN9ZuOFIYGpGB2VYxEBndWYKfBIWu1cUbgDlW7gKAI5XVGflC3vhBgAEpmYElS19qWpoUuhoe9iy5qgWW2E64KCTJN3tpKyzkKXS5qCiry+84z8BPEt3KZY1GmgHDkHJps2AbD/7lLKzKww3iIiIiIiIbrVKz+CwogPwPgCts0IrbgCGArDYJcAZRzf6JtrY6LcNaSkfwX5gc9JzWNzj6pg4m2agzuj37vbKHT5kNYAHHZRc9BwWF6WOidvn4LwF/d7dYu7wIWMATEDpEhp7Mj2HxfVWx8Q5naGhjY1+y5CWYi90+tNzWNzz6pg4my1n6b/J3gwO1YPdUWPG/wEQkPNIL8g6y11VPUfEwCN6ICCUvm2KlixH4ZyvLGqUHcLgNXkCxMBAyMXFMJ7+GyWbt6Dk59WQCyrO7DiDg4iIiIiIbiVTwLEKgMXOH3eQpYGpGYOdFRHdzewFHLcbAw4iIiIiIrqVRAAITM2w2Q72TsFwg4iIiIiIiIicKV/SEZiaISjDImZXVHyLzXa2NIWIyqhUzipuLYUiQ/DyqvamxERERERERI4Isp0mgprwkJEAngcQYHPy5roKYHlgasZcZ4X0/+3bMSvEcQDGcfenLM5lICy8CN1+q5u8gRsMivJ6lEFRt1oMUsdrYXB3GU9dEcdoUb/ljx59PvPzCr71wJfnw+PL18HdXmn3WxbaOyet/vlRaQcAAFCXbwMHkOV9NF6Z9PZPZ/cPf343qzbWr5Yvzg7mt7fGpS0AAEBdBA74Jz6m0+rl+qYzexxulrY/pVpbfVrs7t42mktvpS0AAECdBA4AAAAgXjUHAAAAEE7gAAAAAOIJHAAAAEA8gQMAAACIJ3AAAAAA8QQOAAAAIJ7AAQAAAMQTOAAAAIB4AgcAAAAQT+AAAAAA4gkcAAAAQDyBAwAAAIgncAAAAADxBA4AAAAgnsABAAAAxBM4AAAAgHgCBwAAABBP4AAAAADiCRwAAABAPIEDAAAAiCdwAAAAAPEEDgAAACCewAEAAADEEzgAAACAeAIHAAAAEE/gAAAAAOIJHAAAAEC8TwWyRAxRSHvjAAAAAElFTkSuQmCC"/></svg>
```
Comment (tauthomas01) [inline app/assets/images/payment_icons/touchngo.svg:1]: This one also

image

**Review by tauthomas01 — CHANGES_REQUESTED** [review]
I'm seeing pixelated payment branded icons.
Is it possible to revisit those changes?

**Comment by jawad-eGHL** [comment]
Hi @tauthomas01,
the pixelation has been removed from the icons you listed above.

File: app/assets/images/payment_icons/boost.svg [hunk 3]
```diff
@@ -0,0 +1 @@
...1LAAUAMalA+C0PSaqHtPxBSZmkPXyEwEL7LFkHRqKH1S1rInzQFxStWOTx/owEHERERERHRjahwiYq53OFDDJrwkE+c1VWWLjG+sSY8JN9ZuOFIYGpGB2VYxEBndWYKfBIWu1cUbgDlW7gKAI5XVGflC3vhBgAEpmYElS19qWpoUuhoe9iy5qgWW2E64KCTJN3tpKyzkKXS5qCiry+84z8BPEt3KZY1GmgHDkHJps2AbD/7lLKzKww3iIiIiIiIbrVKz+CwogPwPgCts0IrbgCGArDYJcAZRzf6JtrY6LcNaSkfwX5gc9JzWNzj6pg4m2agzuj37vbKHT5kNYAHHZRc9BwWF6WOidvn4LwF/d7dYu7wIWMATEDpEhp7Mj2HxfVWx8Q5naGhjY1+y5CWYi90+tNzWNzz6pg4my1n6b/J3gwO1YPdUWPG/wEQkPNIL8g6y11VPUfEwCN6ICCUvm2KlixH4ZyvLGqUHcLgNXkCxMBAyMXFMJ7+GyWbt6Dk59WQCyrO7DiDg4iIiIiIbiVTwLEKgMXOH3eQpYGpGYOdFRHdzewFHLcbAw4iIiIiIrqVRAAITM2w2Q72TsFwg4iIiIiIiIicKV/SEZiaISjDImZXVHyLzXa2NIWIyqhUzipuLYUiQ/DyqvamxERERERERI4Isp0mgprwkJEAngcQYHPy5roKYHlgasZcZ4X0/+3bMSvEcQDGcfenLM5lICy8CN1+q5u8gRsMivJ6lEFRt1oMUsdrYXB3GU9dEcdoUb/ljx59PvPzCr71wJfnw+PL18HdXmn3WxbaOyet/vlRaQcAAFCXbwMHkOV9NF6Z9PZPZ/cPf343qzbWr5Yvzg7mt7fGpS0AAEBdBA74Jz6m0+rl+qYzexxulrY/pVpbfVrs7t42mktvpS0AAECdBA4AAAAgXjUHAAAAEE7gAAAAAOIJHAAAAEA8gQMAAACIJ3AAAAAA8QQOAAAAIJ7AAQAAAMQTOAAAAIB4AgcAAAAQT+AAAAAA4gkcAAAAQDyBAwAAAIgncAAAAADxBA4AAAAgnsABAAAAxBM4AAAAgHgCBwAAABBP4AAAAADiCRwAAABAPIEDAAAAiCdwAAAAAPEEDgAAACCewAEAAADEEzgAAACAeAIHAAAAEE/gAAAAAOIJHAAAAEC8TwWyRAxRSHvjAAAAAElFTkSuQmCC"/></svg>
```
[the remark this answers is not in the document]
Comment (tauthomas01) [reply app/assets/images/payment_icons/boost.svg:1]: I still see this icon being pixelated.

File: app/assets/images/payment_icons/touchngo.svg [hunk 4]
```diff
@@ -0,0 +1 @@
...c-.2 0-.4 0-.4-.1-.1 0-.1-.1-.1-.2v-.2l.4-2.1h.6l-.1.6h.5l-.1.5h-.5l-.3.8v.1c0 .1 0 .1.1.1h.4l-.1.5z"/><path class="st5" d="M18.2 15.1L17 17.7h-.8l-.1-1.6-.8 1.6h-.8l-.1-2.6h.5l.1 2.1 1-2.1h.6l.1 2.1 1-2.1zM20.6 17.7h-.5l.5-2.6h.6zM21.6 17.7H21l.5-2.6h.6zM25.4 17.7H25c-.2 0-.4 0-.4-.1-.1 0-.1-.1-.1-.2v-.2l.4-2.1h.6l-.1.6h.5l-.1.5h-.5l-.3.8v.1c0 .1 0 .1.1.1h.4l-.1.5z"/><path class="st5" d="M18.2 15.1L17 17.7h-.8l-.1-1.6-.8 1.6h-.8l-.1-2.6h.5l.1 2.1 1-2.1h.6l.1 2.1 1-2.1zM20.6 17.7h-.5l.5-2.6h.6zM21.6 17.7H21l.5-2.6h.6zM25.4 17.7H25c-.2 0-.4 0-.4-.1-.1 0-.1-.1-.1-.2v-.2l.4-2.1h.6l-.1.6h.5l-.1.5h-.5l-.3.8v.1c0 .1 0 .1.1.1h.4l-.1.5zM14 15.8c-.1 0-.2-.1-.4-.1h-.8c-.2 0-.4 0-.5.1-.2.2-.3.3-.3.5l-.2.8V17.4s0 .1.1.1h1.9l.1-.5h-1.3l-.1-.1v-.1H14l.1-.7v-.2l-.1-.1m-.5.7h-1v-.1s0-.1.1-.1H13.3c.1 0 .2 0 .2.1v.1M20 15.9s-.1-.1 0 0c-.1-.1-.2-.1-.3-.1H18v.4h1.3s.1 0 .1.1v.1h-1.1c-.1 0-.2 0-.2.1-.1 0-.1.1-.1.2s-.1.2-.1.3v.5l.1.1h1.9l.3-1.4V16c-.2 0-.2 0-.2-.1m-1.6 1.3c-.1 0-.1 0-.1-.1V17l.1-.1h.8l-.1.3h-.7zM24.2 15.8c-.1 0-.2-.1-.4-.1H23c-.2 0-.4 0-.5.1-.1.1-.2.2-.2.4l-.3 1V17.5s0 .1.1.1H24l.1-.5h-1.3l-.1-.1v-.1h1.5l.1-.7V16c0-.1 0-.1-.1-.2m-.5.7h-1v-.1s0-.1.1-.1H23.5c.1 0 .2 0 .2.1v.1"/></g></svg>
```
Comment (tauthomas01) [inline app/assets/images/payment_icons/touchngo.svg]: Can we remove all unused `class` and `id` such as `class="st3"` and `pi-SVGID_1_`?

**Review by tauthomas01 — CHANGES_REQUESTED** [review]
The icons look good.
I saw one icon still pixelated and another icon with many unused attributes.

File: app/assets/images/payment_icons/touchngo.svg (hunk 4 above)
Comment (jawad-eGHL) [reply app/assets/images/payment_icons/touchngo.svg]: Hi @tauthomas01 ,
Sorry forgot to include boost icon so I have added it in latest [commit](https://github.com/activemerchant/payment_icons/pull/393/commits/30611929eeb39ff2a3ed062ecf259a013549b8b8) now.
Regarding the attributes in touchngo icon. I tried removing them but it causes icon colors to change. They are not the unused attributes as removing them affects the icon.

**Review by tauthomas01 — APPROVED** [review]
Thank you for the fixes

**Comment by jawad-eGHL** [comment]
@tauthomas01 thanks for the reviews and merging.

**Comment by alfredkho** [comment]
Hi @tauthomas01, would like to follow up on this request. May we know if all the logo has been made available on partner and merchant account? 

Thanks.

**Comment by tauthomas01** [comment]
@alfredkho All icons should be available except `cimbthai` and `dokuwallet` because those files were [breaking the build](https://github.com/activemerchant/payment_icons/pull/397) of the gem so we had to remove them.

**Comment by alfredkho** [comment]
@tauthomas01 thanks for the update. For CIMB Thai and DokuWallet, please refer to https://github.com/activemerchant/payment_icons/pull/420.
Thank you.

**Pull request merged**